SceneMsg scene
ActionMsg[] actions
AgentMsg[] agents
string PDDLDomain
string PDDLProblem
string type
string motionPlannerType